libs/antsi/README.md

```diff
@@ -0,0 +1,48 @@
+[license]: https://github.com/hashintel/hash/blob/main/libs/deer/LICENSE.md
+
+# antsi
+
+`antsi` is a no-std mini-crate that provides support for SGR[^1] codes (better known as ANSI escape sequences)
+
+The name is a play on words and encapsulates the three goals of the crate:
+
+- ant: small, productive and extremely useful (🐜)
+- ansi: implementation of ANSI escape sequences
+- antsy: restless as in fast, with near to no overhead (🏎️💨)
+
+The crate tries to be as correct as possible, with the goal of being not only a crate, but also a comprehensive reference about terminal support and related specifications.
```

Comment:
While I think this is a great goal for the crate, I suggest we seek to "undersell and over-deliver" here, and avoid:

- claiming comprehensiveness
- committing ourselves to ongoing maintenance, making clear that correctness is claimed only at the point of publication (while we likely want to do this, we should gauge the initial reaction to `antsi` first)

```suggestion
The crate tries to be as correct as possible, acting both as a library and as an up-to-date reference guide regarding terminal support and related specifications (correct as of the time of publication).
```